dist/test_data.py

```diff
@@ -41,16 +41,38 @@ def __gt__(self, other):
 
     def __ge__(self, other):
         return self.name >= other.name
+
+key_config=[
+    Config('key_size', 0, r'''
+        The size of the keys to be created''', min=0, max=10000),
+]
+value_config = [
+    Config('value_size', 0, r'''
+        The size of the values to be created''', min=0, max=10000),
+]
+scale_config = [
```

Comment:
I think we should separate config's with newlines. Which would be consistent with `api_data.py`. So space out key_config, value_config, etc.